src/celldega/pre/boundary_tile.py

```diff
@@ -7,6 +6,125 @@
 import geopandas as gpd
 from shapely.geometry import Polygon, MultiPolygon
 
+def numpy_affine_transform(coords, matrix):
```

Comment:
Confirm that we are still able to pre-process Xenium datasets (can use this template https://colab.research.google.com/drive/1guUFhXP3nlZ4Es2-tsnraFKlAKHZSCZC#scrollTo=tyEZ3U9kTw0p) after these changes are made.